misc/scripts/install-local.sh

```diff
@@ -271,6 +271,25 @@ function is_compatible_arch() {
     fi
 }
 
+function is_compatible_cpuflags() {
+    local input=("${@}") ret=0
+    for pcpuflag in "${input[@]}"; do
+        local flag_found=0
+        for dcpuflag in "${DCPUFLAGS[@]}"; do
+            if [[ "$pcpuflag" == "$dcpuflag" ]]; then
+                flag_found=1
+                break
+            fi
+        done
+        if ((flag_found==0)); then
```

Comment:
```suggestion
        if ((flag_found)); then
```

Reply:
Nope. Not correct.

Reply:
Why not.

Reply:
This would be checking for the existence of the variable, not the value of it, 4head.

Reply:
no it wouldn't... `0` evaluates to success and `1` to fail, then put that in an if statement and it's checking exit codes.

Reply:
Except it doesn’t. Try it yourself.

Reply:
![image](https://github.com/pacstall/pacstall/assets/58742515/d2ef0805-f2c9-4e2d-8ace-3d3ddbbb47fa)

Reply:
Fine I concede bash is being stupid.